config/local_config_test.go

```diff
@@ -52,39 +52,43 @@ func TestLoadARMALocalConfigAndCrypto(t *testing.T) {
 	// 3.
 	for i := 1; i <= len(networkLocalConfig.PartiesLocalConfig); i++ {
 		configPath := path.Join(dir, "config", fmt.Sprintf("party%d", i), "local_config_router.yaml")
-		routerLocalConfigLoaded, err := config.LoadLocalConfig(configPath)
+		routerLocalConfigLoaded, role, err := config.LoadLocalConfig(configPath)
```

Comment:
Add a test in which the LoadLocalConfig fails because of multiple roles or no roles, to check the `validateNodeLocalConfigParams`.